src/debug/HyprCtl.cpp

```diff
@@ -895,6 +895,12 @@ std::string systemInfoRequest(eHyprCtlOutputFormat format, std::string request)
         result += std::format("  {} by {} ver {}\n", pl->name, pl->author, pl->version);
     }
 
+    if (g_pHyprCtl->m_sCurrentRequestParams.sysInfoConfig) {
+        result += "\n======Config-Start======\n";
+        result += "Config: " + execAndGet(std::string("cat " + g_pConfigManager->configCurrentPath).c_str()) + "\n";
```

Comment:
needs to handle sourced files too

Reply:
Yea, my hope was originally that I could just dump the in memory config easily, but it looks like there isn't a clean way to do that without checking a ton of different fields and objects that the different config is stored into.

I'll probably just keep all parsed config lines saved in a buffer on the ConfigManager object so that its easy to print out when this is called.

Lmk if you have a better way.

Reply:
you can `cat` all files in the stat map in cconfigmanager

Reply:
Done!